code/artificial_intelligence/src/linear_classifier/Least_square_classifier/least_square_classifier.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import matplotlib.pyplot as plt
+import pylab as pl
+import numpy as np
+from pylab import *
+
+def LeastSquareClassifier(inputData):
```

Comment:
You really need to increase the indentation here. Otherwise this is hard to read. Check the style guide.